src/Admin/Settings.php

```diff
@@ -89,6 +90,13 @@ public function getSettings()
                 'type' => 'checkbox',
                 'desc' => 'Automatically send orders to Penny Black',
             ),
+            self::FIELD_TRANSMIT_ON_STATUSES => array(
+                'id' => self::FIELD_TRANSMIT_ON_STATUSES,
+                'name' => 'Transmit on order statuses',
+                'type' => 'multiselect',
+                'options' => \wc_get_order_statuses(),
+                'desc' => 'Ctrl-click to select the statuses that will trigger order transmission (default is Pending, On Hold and Processing)',
```

Comment:
or Cmd-click on mac?

Reply:
maybe I'll just remove that part... it's long enough already